src/server/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 pkg_check_modules(FREETYPE freetype2 REQUIRED IMPORTED_TARGET) # For rendering decorated window titles
+pkg_check_modules(PIXMAN REQUIRED IMPORTED_TARGET pixman-1)
```

Comment:
You will need to add the pixman dev package as appropriate in:
- `debian/control`
- `spread/build/*/task.yaml`